src/fondant/executor.py

```diff
@@ -212,9 +212,9 @@ def upload_manifest(self, manifest: Manifest, save_path: t.Union[str, Path]):
 
         if is_kubeflow_output:
```

Comment:
I'm not the biggest fan of having kubeflow specific code in here. Opened #376 to remove this.

Reply:
I agree, though I don't see a clear-cut solution for this just yet

Reply:
will vertex have the same issue ?

Reply:
probably, ideally we have some control over the format and save path of the artifact.